spec/controllers/slides_controller_spec.rb

```diff
@@ -61,6 +70,28 @@
     end
   end
 
+  describe 'POST #create' do
+    it 'succeed to create record' do
+      allow_any_instance_of(SlidesController).to receive(:send_message).and_return(true)
+      data = build(:slide)
+      login_by_user first_user
+      post :create, slide: data.attributes
+      id = Slide.where(:key => data.key).first.id
```

Comment:
__[rubocop]__
* Style/HashSyntax: Use the new Ruby 1.9 hash syntax. [:ref](https://github.com/bbatsov/ruby-style-guide#hash-literals)